<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: Friday, December 6, 2024 8:19:12 PM</msg><msg code="0" type="0">Con("Classified_Bangladesh_Composite4.tif" != 2, "Landsat_Composite_AllBands_Bang.tif")</msg><msg code="539" type="100">ERROR 000539: Traceback (most recent call last):
  File "&lt;expression&gt;", line 1, in &lt;module&gt;
  File "&lt;string&gt;", line 5, in rcexec_DC684390_0C54_4F14_9EBF_B34DF3F2E1B1
  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\sa\Functions.py", line 2759, in Con
    return Wrapper(
  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\sa\Utils.py", line 55, in swapper
    result = wrapper(*args, **kwargs)
  File "C:\Program Files\ArcGIS\Pro\Resources\ArcPy\arcpy\sa\Functions.py", line 2746, in Wrapper
    return _wrapLocalFunctionRaster(u"Con_sa",
RuntimeError: ERROR 000732: Input Raster: Dataset Landsat_Composite_AllBands_Bang.tif does not exist or is not supported
</msg><msg code="-2147467259" type="100">Failed to execute (RasterCalculator).</msg><msg code="0" type="3">Failed at Friday, December 6, 2024 8:19:12 PM (Elapsed Time: 0.32 seconds)</msg></messages>